index.js

```diff
@@ -44,13 +44,13 @@ for (let i = 0; i < datasets.length; i++) {
     var card =  `<div class="card" id="dataset-${i}">` +
                     `<div class="card-header" id="heading-${i}">` +
                         '<h6 class="mb-0">' + 
-                            `<a class="collapsed text-left stretched-link" type="button" data-target="#collapse-${i}" aria-expanded="false" aria-controls="collapse-${i}" onclick="toggleMetadata(${i}, false);">` +
+                            `<a class="collapsed text-left stretched-link" data-target="#collapse-${i}" aria-expanded="false" aria-controls="collapse-${i}" onclick="toggleMetadata(${i}, false);">` +
```

Comment:
Thanks for catching that! In Firefox it doesn't change anything, but chrome looks horrible with `type="button"`!